be/src/storage/lake/update_manager.cpp

```diff
@@ -110,6 +114,9 @@ Status UpdateManager::publish_primary_key_tablet(const TxnLogPB_OpWrite& op_writ
     for (const auto& one_delete : state.deletes()) {
         index.erase(*one_delete, &new_deletes);
     }
+    for (const auto& one_delete : auto_increment_deletes) {
```

Comment:
If we can hide `fetch_conflict_deletes_by_auto_increment` in `state->load`, then `auto_increment_deletes` can be replace by `state.auto_increment_deletes()`.

Reply:
done